typegate/src/config.ts

```diff
@@ -39,6 +39,9 @@ const schema = {
     }
     return bytes;
   }),
+  timer_max_timeout_ms: z.number().positive().max(60000),
+  timer_max_timeout_test_ms: z.number().positive().max(60000),
```

Comment:
I was more thinking on injection config.timer_max_timeout_ms in utils/mod.ts because adding an env var just for test sounds a bit heavy :)